docs/installation/pipx-based.rst

```diff
@@ -124,11 +154,19 @@ To install ``pyenv``, follow these steps:
 
     Then, complete the setup by adding ``pyenv`` to your profile, so the executable can be found. `Check the instructions in the official webpage <https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv>`_.
 
-- **Windows**: ``pyenv-win`` is a separate project but it has the same functionality and it is also simpler to setup. Just run the following command and you should be ready to go:
+- **Windows**: ``pyenv-win`` is a separate project but it has the same functionality and it is also simpler to setup.
+You can read the detailed installation instructions `from the official pyenv-win website <https://github.com/pyenv-win/pyenv-win/tree/master>`_,
+but the easiest way is to run the following command in the ``powershell`` and, upon closing and launching a new shell, you should be ready to go:
 
     .. code-block:: powershell
 
-        Invoke-WebRequest -UseBasicParsing -Uri "https://raw.githubusercontent.com/pyenv-win/pyenv-win/master/pyenv-win/install-pyenv-win.ps1" -OutFile "./install-pyenv-win.ps1"; &"./install-pyenv-win.ps1"
+        # Step 1: In your powershell, invoke the following command:
+        `Invoke-WebRequest -UseBasicParsing -Uri "https://raw.githubusercontent.com/pyenv-win/pyenv-win/master/pyenv-win/install-pyenv-win.ps1" -OutFile "./install-pyenv-win.ps1"; &"./install-pyenv-win.ps1"`
+
+        # Step 2: close the shell you invoked in the command from Step 1 and re-launch an new `powershell`. This is equivalent to invoke the shell command `source` in Linux.
+
+        # Step 3: Confirm success; you should be returned something similar to `pyenv 3.1.1`
+        `pyenv --version`
```

Comment:
A few things:

1.  It's already a code block, so if you put backticks inside it then Sphinx will render them, i.e.:

![Screenshot from 2024-07-02 at 09_14_02 795107487](https://github.com/EnergySystemsModellingLab/MUSE_OS/assets/23149834/26a23bd2-926c-411b-a045-73bb7d4466b6)

2. Relaunching powershell isn't exactly the same as running `source` in bash, so I've just removed that comment

```suggestion
        # Step 1: In your powershell, invoke the following command:
        Invoke-WebRequest -UseBasicParsing -Uri "https://raw.githubusercontent.com/pyenv-win/pyenv-win/master/pyenv-win/install-pyenv-win.ps1" -OutFile "./install-pyenv-win.ps1"; &"./install-pyenv-win.ps1"

        # Step 2: close the shell you invoked in the command from Step 1 and re-launch powershell

        # Step 3: Confirm success; you should be returned something similar to `pyenv 3.1.1`
        pyenv --version
```

Reply:
Great. I can commit this suggestion. 

Thanks for pointing out that invoking `source` on on the Linux Command Line is not equivalent to re-launching a powershell on Windows. 

I thought about this more deeply and came up with the following:
`source` executes a script within the current shell session, preserving environment variables and other session state changes. Relaunching PowerShell starts a new session, losing any changes made in the previous session. In addition, `source` runs scripts in the same process, while relaunching PowerShell creates a new process.

I won't mention this in the documentation, but are my thoughts correct?

Reply:
Yeah, exactly :+1: